src/main/java/com/openlattice/chronicle/api/ChronicleApi.java

```diff
@@ -27,6 +25,8 @@
     String DATASOURCE_ID   = "datasourceId";
     String DATE            = "date";
     String ENTITY_KEY_ID   = "entityKeyId";
+    String MESSAGE_ID      = "MessageSid";
+    String MESSAGE_STATUS  = "MessageStatus";
```

Comment:
These are query string params and should follow the conventions as with variable names (like how you documented it below), i.e.

```
?messageSid=123&messageStatus=success
```

```suggestion
    String MESSAGE_ID      = "messageSid";
    String MESSAGE_STATUS  = "messageStatus";
```

Reply:
will this be an issue if the body of the status looks like this:
```
SmsSid: SM2xxxxxx
SmsStatus: sent
MessageStatus: sent
To: +1512zzzyyyy
MessageSid: SM2xxxxxx
AccountSid: ACxxxxxxx
From: +1512xxxyyyy
ApiVersion: 2010-04-01
```

Reply:
It would be helpful to understand how that data interacts with the frontend/backend. Where does this show up? Where is it used?

The change I'm proposing means the frontend will make the following request
```
https://api.openlattice.com/chronicle/v2/.../message/status?messageSid=SM2xxxxxx&messageStatus=sent
```
The value for these params gets passed to the variables in the controller, so

```
public void updateMessageStatus(
  @PathVariable( ORGANIZATION_ID ) UUID organizationId,
  @RequestParam( MESSAGE_SID ) String messageSid,
  @RequestParam( MESSAGE_STATUS ) String messageStatus
) {
  // messageSid is set to "SM2xxxxxx"
  // messageStatus is set to "sent"
```

Reply:
This is not an API that we are hitting from the front end. This is the status callback URL for Twilio. So when Twilio receives a delivery status, it hits this API with that status. The body of that response looks like the above.

Reply:
confirmed that it needs to be `MessageSid` and `MessageStatus`.